packages/backend-core/src/security/roles.ts

```diff
@@ -7,18 +7,11 @@ import {
   doWithDB,
 } from "../db"
 import { getAppDB } from "../context"
-import { Screen, Role as RoleDoc } from "@budibase/types"
+import { Screen, Role as RoleDoc, Roles } from "@budibase/types"
```

Comment:
For enums we follow the rule of the enumeration names being singular, this can sometimes read a little bit better, e.g. `Options.One` is a bit weirder than `Option.One`

Reply:
Renamed to `BuiltInRole` to avoid conflict with the `Role` document type.